packages/devextreme/js/__internal/ui/calendar/m_calendar.ts

```diff
@@ -744,6 +745,17 @@ const Calendar = Editor.inherit({
     }
   },
 
+  _updateAriaReadonly(readonly = false) {
+    const element = this.$element();
+    if (!readonly) {
+      element.removeAttr('role');
+      element.removeAttr('aria-label');
+    } else {
+      element.attr('role', 'group');
+      this.setAria('label', 'Read-only calendar', element);
```

Comment:
[missing scenario]
Kindly add a localization for this string

Reply:
[28bc567](https://github.com/DevExpress/DevExtreme/pull/27898/commits/28bc5675e82786dfe12c0434627ece5fe383cf65)